src/Twig/TokenParser/CellParser.php

```diff
@@ -38,6 +38,14 @@ class CellParser extends IncludeTokenParser
      */
     public function parse(Token $token): Node
     {
+        static $warned = false;
+        if (!$warned) {
+            $warned = true;
+            deprecationWarning(
+                '`cell` and `element` tags are deprecated. Use `cell()` and `element()` functions instead.'
```

Comment:
`CellParser` class should only warn about `cell` tag :slightly_smiling_face:

Reply:
done.